pom.xml

```diff
@@ -29,7 +29,7 @@
     <version.org.mortbay.jetty>6.1.25</version.org.mortbay.jetty>
     <version.mysql>5.1.35</version.mysql>
     <version.postgresql>9.1-901-1.jdbc4</version.postgresql>
-    <version.com.fasterxml.jackson.core>2.7.2</version.com.fasterxml.jackson.core>
+    <version.com.fasterxml.jackson.core>2.9.9.3</version.com.fasterxml.jackson.core>
```

Comment:
I think you need to split the property as core is 2.9.9, and jackson-databind on 2.9.9.3

Reply:
splitted!